packages/synchro-charts/cypress/integration/charts/scatter-chart/sc-webgl-scatter-chart.spec.ts

```diff
@@ -113,4 +113,14 @@ describe('scatter chart', () => {
 
     cy.matchImageSnapshotOnCI();
   });
+
+  it('renders an error when the dataStream has a non-numerical dataType', () => {
+    cy.visit(`${root}/unsupported-data-types`);
+
+    cy.waitForChart();
+
+    cy.get('.unsupported-data-type-status').contains('NUMBER');
+
+    cy.matchImageSnapshotOnCI();
```

Comment:
how much value do you think this snapshot is adding? Not a huge fan of snapshots if we can avoid them.

Reply:
yeah not much in my opinion, will just check for the div+text being there instead